jaxrs/pom.xml

```diff
@@ -107,29 +107,6 @@
             <artifactId>kafka-cdi-extension</artifactId>
```

Comment:
The consumer is in the sender module. No connection with jaxrs if I am not wrong.